### ISSUE TITLE
Trouble with downloading universal packages

### ISSUE THREAD

**Issue by eedpWillVinke** [Microsoft/vsts-cli#237]
I am trying to use you're vsts-cli 0.1.3 tool to download a universal package as part of a build pipeline. I am running into trouble getting the `vsts package universal download` command to work. When the script gets to this part, it stays on 0kb and does not seem to progress with the download on certain machines, but downloads with no issue on the build agents and my local machine.

Could this be an issue with Artifacts licencing? I thought it was possible to only download an artifact without the full licence. Would it be an issue with throttling in some way?

The following command is the one that's been giving me trouble, but the problem doesn't seem to be from the command syntax. 
`vsts package universal download --feed "RPackages" --instance $organization --name "r-3.5.0" --version "1.0.4" --path $destPath #--debug`

Thanks,
     Will

**Comment by zarenner** [comment]
Hi @eedpWillVinke,

Are you able to share the output of the debug log? It's unclear to me if it's getting stuck while downloading the "client tools" (a tool called artifacttool that vsts package universal uses under the covers), or if it's getting stuck on the actual download of the package.

I don't think it's licensing related (I'd expect an error, not a hang), but if it's an issue with downloading the client tools, it could potentially be an issue with the CDN hosting the tools. It could also be a problem with invoking the downloaded client tools, although with that too I would expect a failure rather than a hang.

In short, a bit more detail would help and I'd be happy to look into this.

**Comment by eedpWillVinke** [comment]
Hey @zarenner,

Sorry about the slow response. I have been trying to reproduce this problem one of my team members saw but with no luck, leading me to think it may have been something he was doing locally such as accidentally pausing powershell or some other dumb issue. It only happened one time, to one person so I'm not sure what the issue was and it does not seem to be one anymore. 

We have gone ahead and used your tool in our main branch. It has worked perfectly so far except for this one glitch. If another issue arises, the debug output is always on so we will have more info. 

Thanks again for you support, sorry I couldn't help you get to the bottom of this.
